ddsp/training/docker/user_interaction.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+"""Script for gathering input, building the image and submitting job"""
+
+import os
+
+def get_input():
+  """Gathers input from user"""
+  # Determining path for data retrieval
+  input_dict = dict()
+  data_path = input('Insert a path for data retrieving. '
```

Comment:
it might be nice to add some verification of user inputs, for instance that they start with `gs://` and if not try again and tell them what they did wrong. Similarly, it might be possible to do a quick call to google.cloud.storage to see if you can access the bucket before actually running the program